app/jobs/scheduled/pending_users_reminder.rb

```diff
@@ -12,7 +12,7 @@ def execute(args)
           query = query.where('users.created_at < ?', SiteSetting.pending_users_reminder_delay.hours.ago)
         end
 
-        newest_username = query.limit(1).pluck(:username).first
+        newest_username = query.limit(1).select(:username).first&.username
```

Comment:
Fast pluck doesn't seem to work with subqueries:

     Failure/Error: construct_relation_for_association_calculations.pluck(*cols)
     
     NoMethodError:
       undefined method `empty?' for #<Arel::Nodes::TableAlias:0x007ffaa4b27338>
     # ./lib/freedom_patches/fast_pluck.rb:59:in `pluck'
     # ./app/jobs/scheduled/pending_users_reminder.rb:15:in `execute'
     # ./spec/jobs/pending_users_reminder_spec.rb:13:in `block (3 levels) in <top (required)>'